Resources/Prototypes/Entities/Structures/Machines/mining.yml

```diff
@@ -0,0 +1,19 @@
+- type: entity
+  id: Electromagnet
+  parent: BaseMachinePowered
+  name: electromagnet
+  description: "Generates a magnetic field when powered."
+  components:
+  - type: Sprite
+    layers:
+      - sprite: Structures/Machines/electromagnet.rsi
+        state: icon
+  - type: Rotatable
+  - type: Transform
+    noRot: false
```

Comment:
noRot shouldn't be here if it has rotatablecomponent.

Reply:
Taking away `noRot: false` makes this non-rotatable even though it has a `Rotatable` component.